pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
The _test suffix is actually a go convention. It has a special meaning, that we're going to test the public interface only. Take a look, for example, at the standard library strings package: https://golang.org/src/strings/search_test.go